kubetest/kops.go

```diff
@@ -483,3 +527,93 @@ func (k *kops) runKopsDump() (*kopsDump, error) {
 
 	return dump, nil
 }
+
+// getRandomAWSZones looks up all regions, and the availability zones for those regions.  A random
+// region is then chosen and the AZ's for that region is returned.  If HA is required a regions with 3
+// or more zones is returned.
+func getRandomAWSZones(isHA bool) (*[]string, error) {
```

Comment:
So I will counterpose something.  How about we have a blacklist of regions, so that when US-Colorado opens we blacklist it?  Also, when a region gets its third AZ it will automatically start running HA tests.  I would rather know which regions are having issues.  Just a thought.